core/kazoo_tasks/Makefile

```diff
@@ -3,4 +3,12 @@ PROJECT = kazoo_tasks
 
 all: compile
 
+test: check_csvs
+
+check_csvs:
+	@stat -t -- test/*.csv >/dev/null 2>/dev/null ; if [[ $$? > 0 ]] ; then echo WARNING: run 'make csvs' for more test data ; fi
```

Comment:
How big is this set of csv files? Does it get updated often? How about commiting it to Kazoo?